draft-irtf-cfrg-vdaf.md

```diff
@@ -3074,8 +3080,47 @@ def decode(self, output, _num_measurements):
     return [bucket_count.as_unsigned() for bucket_count in output]
 ~~~
 
-The `Histogram` validity circuit uses `Range2` (see {{prio3sum}}) as its single
-gadget. It checks for one-hotness in two steps, as follows:
+This validity circuit uses a `ParallelSum` gadget to achieve a smaller proof
+size. This optimization for "parallel-sum circuits" is described in
+{{BBCGGI19}}, section 4.4. Briefly, for circuits that add up the output of
+multiple identical subcircuits, it is possible to achieve smaller proof sizes
+(on the order of O(sqrt(n)) instead of O(n)) by packaging more than one such
```

Comment:
nit: Since `n` is undefined :)
```suggestion
(on the order of O(sqrt(length)) instead of O(length)) by packaging more than one such
```

Reply:
I changed this to `MEAS_LEN` instead, now that this is moving back to Prio3SumVec.